projects/hslayers/src/components/utils/layer-utils.service.ts

```diff
@@ -191,12 +193,58 @@ export class HsLayerUtilsService {
   }
 
   /**
-   * @ngdoc method
-   * @name HsLayerUtilsService#isLayerInManager
-   * @param {Ol.layer} layer Layer to check
-   * @returns {boolean} True if showInLayerManager attribute is set to true
-   * @description Test if layer is shown in layer switcher
+   * Test if the features in the vector layer come from a GeoJSON source
```

Comment:
I would might try testing if layer is vector layer with utils function and then use switch case to get the format to make code shorter, but I don't know if there is a special need for this format.

Reply:
Good point. I generally wanted to keep the same logic of boolean-ish functions. That was the only reason really. Your proposal would be more concise of course.

Reply:
I have refactored. It is not exactly as you meant (if I am correct), but it's at least shorter.